Annotato/Annotato/Persistence/PersistenceService/OnlinePersistenceService/OnlinePersistenceService+AnnotationsPersistence.swift

```diff
@@ -14,6 +14,10 @@ extension OnlinePersistenceService: AnnotationsPersistence {
         await remotePersistence.annotations.deleteAnnotation(annotation: annotation)
     }
 
+    func createOrUpdateAnnotation(annotation: Annotation) -> Annotation? {
+        localPersistence.annotations.createOrUpdateAnnotation(annotation: annotation)
+    }
+
```

Comment:
Unrelated to this PR but the methods on lines 5, 9 and 13 should be calling remote persistence first and then getting the response and saving it to the local persistence same as `OnlinePersistenceService+DocumentsPersistence`.